concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -3473,34 +3538,32 @@ jobs:
     serial: true
     serial_groups: [smoke-tests]
     plan:
-    - aggregate:
-      - get: paas-cf
-      - get: cf-vars-store
-
     - task: generate-cf-admin-password
       tags: [colocated-with-web]
       config:
         platform: linux
         image_resource: *gov-paas-bosh-cli-v2-image-resource
-        inputs:
-          - name: cf-vars-store
         outputs:
-          - name: rotated-cf-vars-store
+          - name: passwords
+        params:
+          CREDHUB_CLIENT: credhub-admin
+          CREDHUB_SECRET: ((bosh-credhub-admin))
+          CREDHUB_CA_CERT: ((bosh-credhub-ca-cert))
+          CREDHUB_SERVER: "https://((bosh_fqdn)):8844/api"
+          DEPLOY_ENV: ((deploy_env))
         run:
           path: sh
           args:
             - -e
             - -c
             - |
-              sed "s/^cf_admin_password:.*$//" cf-vars-store/cf-vars-store.yml \
-                > rotated-cf-vars-store/rotated-cf-vars-store.yml
-
-              bosh interpolate - --vars-store rotated-cf-vars-store/rotated-cf-vars-store.yml <<EOF
-              ---
-              variables:
-              - name: cf_admin_password
-                type: password
-              EOF
+              (tr -cd '[:alpha:]0-9' < /dev/urandom | head -c32) > passwords/cf-admin-password
```

Comment:
Shouldn't we be using `credhub generate` for this?

Reply:
Good point, initially did that so I could pass the output. I'll update it to generate the password then get the value and output it.